docs/en/02_Developer_Guides/14_Files/02_Images.md

```diff
@@ -200,13 +200,112 @@ SilverStripe\Core\Injector\Injector:
       Quality: 90
 ```
 
-### Lazy Loading
+### Lazy Loading {#lazy-loading}
 
-Images added via the WYSIWYG editor and as objects on templates are [lazy loaded](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) by default.
+Most modern browsers support the ability to "lazy load" images by adding a `loading="lazy"` attribute 
+to the `<img />` tag. This defers the loading of images not in the viewport to improve the initial 
+page load performance.
 
-To disable lazy loading for an individual image in a template, use `$MyImage.LazyLoad(false)`.
+Silverstripe CMS automatically adds the `loading="lazy"` to images added in an HTML editor field
+and to images rendered via a SS template file.
 
-To out of lazy loading globally, notably if you already have a custom lazy loading implementation, use the following yml config:
+Read [Browser-level image lazy-loading for the web](https://web.dev/browser-level-image-lazy-loading/)
+on _web.dev_ for more information.
```

Comment:
Before digging into code sample, I wanted to give folks a high level overview of what lazy loading is.